pom.xml

```diff
@@ -671,8 +671,8 @@
                   <goal>exec</goal>
                 </goals>
                 <configuration>
-                  <executable>${java.home}/bin/java</executable>
-                  <commandlineArgs>-cp target/classes:target/test-classes:${test.classpath} org.openjdk.jmh.Main ${jmh.args}</commandlineArgs>
+                  <executable>${java.home}${file.separator}bin${file.separator}java</executable>
+                  <commandlineArgs>-cp "target${file.separator}classes${path.separator}target${file.separator}test-classes${path.separator}${test.classpath}" org.openjdk.jmh.Main ${jmh.args}</commandlineArgs>
```

Comment:
This command line does not look very elegant, due to the fact that:

- we **want** `${jmh.args}` to be split into words.
- we do **not** want the classpath to be split into words.

What would you say about passing the classpath in the `CLASSPATH` environment variable:

```xml
<environmentVariables>
  <CLASSPATH>target${file.separator}classes${path.separator}target${file.separator}test-classes${path.separator}${test.classpath}</CLASSPATH>
</environmentVariables>
```

Reply:
Sure, this seems to work as well. I changed it.

But, I thought that the quotes were already preventing the splitting of the classpath. The JMH args were left unquoted, same as before.

Reply:
> But, I thought that the quotes were already preventing the splitting of the classpath. The JMH args were left unquoted, same as before.

The quotes should work in most normal cases, although I suspect that using `"` in the path of the Maven local repository on UNIX might break the configuration (the plugin uses [`plexus-utils`](https://github.com/codehaus-plexus/plexus-utils/blob/483227443f87a89d61f66d6a4747748bce45fdec/src/main/java/org/codehaus/plexus/util/cli/CommandLineUtils.java#L348) to split the command line).
Using the environment variable is probably just safer.